### PR TITLE
Release Candidate v1.2.5

### PR DESCRIPTION
**Pull request by AlmightyMikkel**

### CODE REVIEW

File: Runtime/Game/LootLockerSDKManager.cs [hunk 1]
```diff
@@ -2121,6 +2164,292 @@ public static void RegisterPlayerProgression(string progressionKey, Action<LootL
...
+
+            data.asset_instance_id = assetInstanceID;
+            data.hero_id = heroID;
+
+
+            LootLockerAPIManager.AddAssetToHeroLoadout(data, onComplete);
+        }
+
+        /// <summary>
+        ///
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:2412]: Missing summary

File: Runtime/Game/LootLockerSDKManager.cs [hunk 2]
```diff
@@ -2121,6 +2164,292 @@ public static void RegisterPlayerProgression(string progressionKey, Action<LootL
...
+
+            LootLockerAddAssetVariationToHeroLoadoutRequest data = new LootLockerAddAssetVariationToHeroLoadoutRequest();
+
+            data.hero_id = heroID;
+            data.asset_id = assetID;
+            data.asset_variation_id = assetInstanceID;
+
+            LootLockerAPIManager.AddAssetVariationToHeroLoadout(data, onComplete);
+        }
+
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:2434]: Missing summary, params etc.

File: Runtime/Game/LootLockerSDKManager.cs [hunk 3]
```diff
@@ -2162,6 +2491,16 @@ public static void ListCharacterTypes(Action<LootLockerListCharacterTypesRespons
             LootLockerAPIManager.ListCharacterTypes(onComplete);
         }
 
+        public static void ListPlayerCharacters(Action<LootLockerPlayerCharacterListResponse> onComplete)
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:2494]: Missing summary, params etc.

File: Runtime/Game/Requests/AssetRequest.cs [hunk 4]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using LootLocker.Requests;
 using System.Linq;
+using UnityEngine.UI;
```
Comment (JohannesLoot) [inline Runtime/Game/Requests/AssetRequest.cs:5]: This using statement is not necessary I believe?

**Review by JohannesLoot — DISMISSED** [review]
Missing summary, params etc. on a few new functions. 
Looks good otherwise!

File: Runtime/Game/LootLockerSDKManager.cs (hunk 1 above)
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:2412]: Fixed

File: Runtime/Game/LootLockerSDKManager.cs (hunk 2 above)
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:2434]: Fixed

File: Runtime/Game/LootLockerSDKManager.cs (hunk 3 above)
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:2494]: Fixed

File: Runtime/Game/Requests/AssetRequest.cs (hunk 4 above)
Comment (kirre-bylund) [reply Runtime/Game/Requests/AssetRequest.cs:5]: Correct, fixed

**Review by kirre-bylund — APPROVED** [review]

**Pull request merged**